test/regress/cli/regress0/printer/issue9928.smt2

```diff
@@ -0,0 +1,9 @@
+(set-logic QF_UF)
```

Comment:
This test is not clear to me -- what would it output before this PR? That is, how does it exhibit the problem? Shouldn't it have, at least a `print_benchmark` command? And maybe some SCRUBBER filtering all the correct lines?

Reply:
The `dump` tester is run on all benchmarks, which relies on print_benchmark and then reparsing.  This PR fixes the issue.